advance-exercises/src/components/03-HigherOrder.js

```diff
@@ -1,29 +1,47 @@
-/*
-  Exercise:
-
-  Make `withMouse` a "higher-order component" that sends the mouse position
-  to the component as props (hint: use `event.clientX` and `event.clientY`).
-
-*/
+/* eslint-disable react/jsx-filename-extension */
+/* eslint-disable react/no-multi-comp */
 import React from 'react';
 import PropTypes from 'prop-types';
 
+
 function withMouse(Component) {
-  return Component;
+  return class comp extends React.Component {
+    constructor(props) {
+      super(props);
+      this.state = { x: null, y: null };
```

Comment:
Adding empty newlines would make the code more readable. In this case before and after variable declarations. What do you think?

Reply:
Thanks for the input. Do you mean something like this - 
`
this.state = {
  x: null,
  y: null,
}
` ?